frontend/src/bundles/common/components/side-bar/libs/components/collapse-button.tsx

```diff
@@ -0,0 +1,45 @@
+import { type ReactElement } from 'react';
+
+import { Box } from '~/bundles/common/components/components.js';
+
+type Properties = {
+    bg?: string;
+    color?: string;
+    handleClick?: () => void;
+    icon: ReactElement;
+    isCollapsed: boolean;
+    label: string;
+};
+
+const CollapseButton = ({
+    bg = 'none',
+    color = 'white',
+    handleClick = (): void => {},
+    icon,
+    isCollapsed,
+    label,
+}: Properties): JSX.Element => {
```

Comment:
```suggestion
const CollapseButton = ({
    label,
    icon,
    isCollapsed,
    bg = 'none',
    color = 'white',
    handleClick = (): void => {},
}: Properties): JSX.Element => {
```
Required properties should be passed first.